update.py

```diff
@@ -487,46 +524,11 @@ def fix_site_shortcode(matchobj):
 
 def logmatch_code(matchobj, prefix):
 
-    try:
-        print("%s m0: %s" % (prefix, matchobj.group(0)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m0" % prefix)
-
-    try:
-        print("%s m1: %s" % (prefix, matchobj.group(1)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m1" % prefix)
-
-    try:
-        print("%s m2: %s" % (prefix, matchobj.group(2)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m1" % prefix)
-
-    try:
-        print("%s m3: %s" % (prefix, matchobj.group(3)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m3" % prefix)
-
-    try:
-        print("%s m4: %s" % (prefix, matchobj.group(4)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m4" % prefix)
-    try:
-        print("%s m5: %s" % (prefix, matchobj.group(5)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m5" % prefix)
-    try:
-        print("%s m6: %s" % (prefix, matchobj.group(6)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m6" % prefix)
-    try:
-        print("%s m7: %s" % (prefix, matchobj.group(7)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except 7" % prefix)
-    try:
-        print("%s m8: %s" % (prefix, matchobj.group(8)))
-    except Exception:  # FIXME: narrow exception type
-        print("%s: except m8" % prefix)
+    for i in range(9):
+        try:
+            print("%s m%d: %s" % (prefix, i, matchobj.group(i)))
+        except IndexError:  # The object has less groups than expected
+            print("%s: except m%d" % (prefix, i))
```

Comment:
Maybe update to f-strings while doing this?

Reply:
done